### PR TITLE
Do not run `zypper inr` after migration from SLE 12

### PR DESCRIPTION
**Pull request by mnowaksuse**
Recently added `zypper inr` triggers bsc#1127212 for migrations where
the base system is SLE 12 (SLES 12 SP4 in our case), does not happen for
migration from SLES 15 base.

**Failures**
_Migration_
* `offline_sles12sp4_media_base_def_full`: https://openqa.suse.de/tests/2512683#step/yast2_i/6
* `offline_sles12sp4_media_tcm_all_full`: https://openqa.suse.de/tests/2511934#step/yast2_i/6
* `offline_sles12sp4_media_base_all_full`: https://openqa.suse.de/tests/2511924#step/yast2_i/6

_Hyper-V_
* `media_upgrade_sles12sp4_allpatterns_hyperv`: https://openqa.suse.de/tests/2512513#step/yast2_i/7
* `media_upgrade_sles12sp4_hyperv`: https://openqa.suse.de/tests/2515420#step/yast2_i/7
* `proxyscc_upgrade_sles12sp4_allpatterns_hyperv`: https://openqa.suse.de/tests/2512514#step/yast2_i/7
* `proxyscc_upgrade_sles12sp4_hyperv`: https://openqa.suse.de/tests/2512546#step/yast2_i/7
* `proxyscc_upgrade_sles12sp4_hyperv`: https://openqa.suse.de/tests/2512534#step/yast2_i/7

**Validation**
* `proxyscc_upgrade_sles12sp4_hyperv`: http://nilgiri.suse.cz/tests/413
* `offline_sles12sp4_media_base_def_full`: http://nilgiri.suse.cz/tests/414

### CODE REVIEW

**Comment by mnowaksuse** [comment]
@coolgw @hjluo If some of you from migration group could review this, please.

File: tests/console/yast2_i.pm [hunk 1]
```diff
@@ -25,7 +25,9 @@ sub run {
 
     select_console 'root-console';
 
-    zypper_call "-i inr";    # install recommended drivers bsc#953522
+    # Install recommended drivers bsc#953522 unless when migrating from
+    # versions before 15 (bsc#1127212)
+    zypper_call "-i inr" unless is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION', '15'));
```
Comment (rwx788) [inline tests/console/yast2_i.pm]: We have another call on L#33 for jeos. Will it make sense to combine both calls?

Comment (mnowaksuse) [reply tests/console/yast2_i.pm]: @ccret Does JeOS need the extra `zypper inr` call, or is the first one enough?

Comment (ccret) [reply tests/console/yast2_i.pm]: @mnowaksuse @rwx788 Calling it one time for JeOS is enough.

**Comment by coolgw** [comment]
If this is an issue we need using record_soft_failure flag out this workround.

**Comment by mnowaksuse** [comment]
Good idea @coolgw, implemented.

Validation runs were re-run.

**Pull request merged**